README.md

```diff
@@ -468,7 +468,7 @@ Setup minikube with assisted-installer (kube-api enabled)
 ```bash
 export PULL_SECRET=<your pull secret>
 # This is a temporary hack - MGMT-8265
-export DISABLED_HOST_VALIDATIONS=api-vip-connected
+export DISABLED_HOST_VALIDATIONS=ignition-downloadable
```

Comment:
Since you fixed MGMT-8265, and after I updated the agent to decode the cert these lines are no longer necessary.
I validated it locally and it works without it.
Let's remove it

Reply:
Ack